pandera/checks.py

```diff
@@ -286,3 +288,298 @@ def __repr__(self):
         name = getattr(self.fn, '__name__', self.fn.__class__.__name__)
         return "<Check %s: %s>" % (name, self.error) \
             if self.error is not None else "<Check %s>" % name
+
+    @staticmethod
+    def greater_than(min_value) -> 'Check':
+        """Create a :class:`Check` ensuring all values of a series are strictly greater
+            than a certain value.
+
+        :param min_value: Lower bound to be exceeded. Must be a type comparable to
+            the dtype of the :class:`pandas.Series` to be validated (e.g. a numerical
+            type for float or int and a datetime for datetime).
+
+        :returns :class:`Check` object
+        """
+        if min_value is None:
+            raise ValueError("min_value must not be None")
+
+        def _greater_than(series: pd.Series) -> pd.Series:
+            """Comparison function for check"""
+            return series > min_value
+
+        return Check(fn=_greater_than, error="greater_than(%s)" % min_value)
+
+    @staticmethod
+    def greater_or_equal(min_value) -> 'Check':
```

Comment:
what do you think of this name? I'm questioning my original naming of it, and wonder if `greater_than_or_equal_to` (similarly `less_than_or_equal_to`) would be more readable

Reply:
After some research I changed it now to `greater/less_than_or_equal_to`. It is very long but at least it is very clear and consistent with `greater/less_than`.

Alternatives would have been:
- as it was (`greater_or_equal`) which is grammatically not perfect and apparently less common throughout the internet.
- very short form ('le', 'ge', ...) as for example in [pandas](https://pandas.pydata.org/pandas-docs/version/1.0.0/reference/api/pandas.DataFrame.ge.html). But that's so short that the names are not self-explanatory and also that the methods might be overseen completely. 

So, the long form you suggested seems to be the best compromise. I dislike it the least ;-)

Reply:
cool! yes, `less_than_or_equal_to` is pushing it in terms of my preference for function name length, but I think explicit is better than acronyms like `le` or `lte` (as I've seen other libraries do it)